android-artifacts.gradle

```diff
@@ -0,0 +1,22 @@
+uploadArchives {
+    /* task androidJavadocs(type: Javadoc) {
+       source = android.sourceSets.main.java.sourceFiles
+   }
+
+   task androidJavadocsJar(type: Jar, dependsOn: androidJavadocs) {
+       classifier = 'javadoc'
+       //basename = artifact_id
+       from androidJavadocs.destinationDir
+   }*/
+
+    task androidSourcesJar(type: Jar) {
+        classifier = 'sources'
+        //basename = artifact_id
+        from android.sourceSets.main.java.sourceFiles
+    }
+
+    artifacts {
+        archives androidSourcesJar
+        //archives androidJavadocsJar
```

Comment:
Does this mean that the JavaDoc doesn't appear when you pull it in as a Gradle dependency like this?